juniper_official/Interfaces/ipv6_Addr.yml

```diff
@@ -0,0 +1,15 @@
+---

```

Comment:
As we discusses move IPv4 and V6 related rules into community

Reply:
moved file to community